rustls/src/error.rs

```diff
@@ -523,7 +524,10 @@ impl fmt::Display for Error {
                 write!(f, "the supplied max_fragment_size was too small or large")
             }
             Self::General(ref err) => write!(f, "unexpected error: {}", err),
+            #[cfg(feature = "std")]
             Self::Other(ref err) => write!(f, "other error: {}", err),
+            #[cfg(not(feature = "std"))]
+            Self::Other(..) => f.write_str("write error"),
```

Comment:
Is this an appropriate description for all cases or should it say something like "unknown error" or "other unspecified error"?